Exercise_Lab05/Exercise_Lab05/Lab5_1.cs

```diff
@@ -0,0 +1,218 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise_Lab05
+{   
+    /// <summary>
+    /// Bài 5.1 trong Lab05
+    /// Yêu cầu:
+    /// Nhập hiện mảng
+    /// Tìm giá trị nhỏ nhất trong mảng
+    /// Đảo các phần tử trong mảng
+    /// Sắp xếp mảng tăng, giảm
+    /// Tìm các số nguyên tố trong Mảng
+    /// Số lượng các sô dương liên tiếp nhiều nhất
+    /// Trung bình cộng các phần tử dương trong mảng
+    /// Mảng có các phần tử âm dương đan xen?
+    /// </summary>
+    class Lab5_1
+    {
+        public void ShowResult()
+        {
+            int[] arr = { 2, 45, -56, 32, 42, 342, -6, 7, 8, -2 };
+            Console.WriteLine("--------3.1----------");
+            Console.WriteLine("Mảng: ");
+            ShowArray(arr);
+            Console.WriteLine();
+            Console.WriteLine("a. Giá trị nhỏ nhất của mảng là: " +Min(arr));
+            Console.Write("b. Mảng sau khi đảo các giá trị: ");
+            ShowArray(ExchangeArray(arr));
+            Console.WriteLine();
+            Console.WriteLine("c. Mảng sau khi sắp xếp: ");
+            Console.Write("Tăng: ");
+            ShowArray(SortIncreaseOrDecrease(arr, true));
+            Console.WriteLine();
+            Console.Write("Giảm: ");
+            ShowArray(SortIncreaseOrDecrease(arr, false));
+            Console.WriteLine();
+            Console.Write("d. Các số nguyên tố trong mảng: ");
+            GetPrime(arr);
+            Console.WriteLine();
+            int[] arr2 = { 2, 45, -56, 32, 42, 342, -6, 7, 8, -2 };
+            Console.WriteLine("e. Số lượng các số dương liên tiếp nhiều nhất là: " + CheckAmountPositive(arr2));
+            Console.WriteLine("f. Trung bình cộng các phần tử dương: "+AvgPositiveItem(arr2));
+            Console.WriteLine("g. Mảng âm dương đan xen? " +(CheckPositiveNegativeIntertwined(arr2) ? "Có" : "Không"));
+        }
+        /// <summary>
+        /// Phương thức hiện mảng
+        /// </summary>
+        public void ShowArray(int[] arr)
+        {
+            foreach(int x in arr)
+            {
+                Console.Write(x + " ");
+            }
+        }
+        /// <summary>
+        /// Phương thức tìm Min trong mảng
+        /// </summary>
+        /// <param name="arr"></param>
+        /// <returns></returns>
+        public int Min(int[] arr)
+        {
+            int min = arr[0];
+            for(int i =0; i< arr.Length; i++)
+            {
+                if(min>arr[i])
+                {
+                    min = arr[i];
+                }
+            }
+            return min;
+        }
+        /// <summary>
+        /// Phương thức đảo mảng
+        /// </summary>
+        /// <param name="arr"></param>
+        /// <returns></returns>
+        public int[] ExchangeArray(int[] arr)
```

Comment:
đảo mảng ở đây e hiểu ý như nào nhỉ

Reply:
là đảo các phần tử trong mảng ạ,  phần tử cuối cùng lên vị trí đầu tiên và phần tử đầu tiên xuống vị trí cuối cùng ạ

Reply:
thế e kiểm tra lại thuật toán có vẻ sai hay sao đó